### PR TITLE
Ruby driver

### PR DESCRIPTION
**Pull request by p-mongo**

### CODE REVIEW

File: integrations/ruby/executor.rb [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+  private
+
+  def set_signal_handler
+    Signal.trap('INT') do
+      @stop = true
+    end
+  end
+
+  def load_data
```
Comment (prashantmital) [inline integrations/ruby/executor.rb:37]: The testData key has to be ignored by the workload executor - see the `testData` key documentation at https://mongodb-labs.github.io/drivers-atlas-testing/spec-test-format.html - I am not sure why you are first deleting and then re-inserting the testData into the cluster here. Is it to make it easier to run locally or to make the script self-sufficient?

Comment (p-mongo) [reply integrations/ruby/executor.rb:39]: I removed the call to this method, with a note to reinstate it when running workload executor by itself.

File: integrations/ruby/executor.rb [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        @error_count += 1
+      end
+      @operation_count += 1
+    end
+  end
+
+  def result
+    {
+      numOperations: @operation_count,
+      numSuccessfulOperations: @operation_count-@error_count-@failure_count,
```
[the remark this answers is not in the document]
Comment (p-mongo) [reply integrations/ruby/executor.rb:117]: This one is for me, as `numSuccesses` is not clear as to what it is referring to (could be number of iterations, for example).

File: integrations/ruby/install-driver.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -ex
+
+gem install --user-install --no-document bson
```
[the remark this answers is not in the document]
Comment (p-mongo) [reply integrations/ruby/install-driver.sh:5]: I add the driver's `lib` subdir to load path in workload executor script. This is all that is needed to use it.

File: integrations/ruby/executor.rb (hunk 1 above)
Comment (prashantmital) [reply integrations/ruby/executor.rb:39]: Great!

File: integrations/ruby/executor.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (prashantmital) [reply integrations/ruby/executor.rb:117]: SGTM

**Pull request merged**